Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM java:8-jre
+FROM openjdk:8-jre
```

Comment:
It uses muscl instead of glibc, and didnt want to take a chance of breaking something. Plus, for downstream consumers, they expect to be able to `apt-get install` stuff for collins to hook into. We could maintain 2 different Dockerfiles that produces a debian and alpine build, but probably outside the scope of this diff